src/components/CampaignContactsChoiceForm.jsx

```diff
@@ -0,0 +1,144 @@
+import type from "prop-types";
```

Comment:
In PR 2 #1370 I do a `git mv` to make the old campaigncontactsform into src/ingest-contact-loaders/csv-upload/react-component.js since it's mostly the csv uploader and also that way the diffs from that (see the PR commits) can be clearly tracked/seen